src/three/TilesRenderer.js

```diff
@@ -615,11 +626,21 @@ export class TilesRenderer extends TilesRendererBase {
 				promise = loader
 					.parse( buffer )
 					.then( res => res.scene	);
-
 				break;
 
 			}
 
+			// 3DTILES_content_gltf
+			case 'gltf':
+			case 'glb':
+				const loader = new GLTFExtensionLoader( manager );
+				loader.workingPath = workingPath;
+				loader.fetchOptions = fetchOptions;
+				promise = loader
+					.parse( buffer )
+					.then( res => res.scene	);
+				break;
```

Comment:
> I think creating a new extension loader class is unnecessary.
Is there a reason to prefer using the base GLTFLoader over the wrapped version? 
1: It's GLTFLoader.parse is a different signature than all of the other existing loaders you have?
2: new GLTFLoader does not support additional gltf extensions/configuration. For instance, we require using basis + draco + a custom gltf extension, and given the underlying worker pools of those, it works better if there is a single instance of the draco/ktx loaders shared across all gltf.parse calls?
3: That also locks into a single working path for the whole tileset with the way things are set up at the moment, as we only have the extension and the buffer available during parse, we do not have the full uri to load at this point?

> fetch options aren't directly supported by three.js
Fair enough, that's not something that we've been using, I had just copied this pattern from the other case entries, should it be removed? the fetchOptions and workingPath are only used during .parse by loaders which reference external uris within the tile? b3dm/gltf? it was already included in the other cases, even though only respected by the .load function, not the parse here?

Reply:
I didn't intend to remove the ability to get a custom GLTF instance via `getHandler` but I'm probably over complicating things. Let's keep it wrapped this way for now and I'll think about it further. Maybe we can revisit another time.

> That also locks into a single working path for the whole tileset with the way things are set up at the moment, as we only have the extension and the buffer available during parse, we do not have the full uri to load at this point?

The `workingPath` is retrieved from the tile geometry URI not the tileset URI. It's extracted from the `tile.content.uri` string [here](https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/master/src/three/TilesRenderer.js#L572-L575).

> the fetchOptions and workingPath are only used during .parse by loaders which reference external uris within the tile? b3dm/gltf? it was already included in the other cases, even though only respected by the .load function, not the parse here?

Both the I3DMLoader and B3DMLoader apply the fetch options to the underlying GLTFLoader options similar to that the PR I linked above (see [here](https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/master/src/three/B3DMLoader.js#L24-L48) and [here](https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/master/src/three/I3DMLoader.js#L41-L65)) (I forgot that was happening). We'll need the same thing happening the GLTFExtensionLoader to make sure the fetch options are applied to the underlying GLTFLoader so different security options and headers are respected. Given that it will be happening in three places now it probably makes sense to create a utility function to apply fetch options to three.js loaders.

https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/master/src/three/B3DMLoader.js#L24-L48